python/whylogs/api/logger/segment_processing.py

```diff
@@ -126,6 +126,7 @@ def segment_processing(
     pandas: Optional[pd.DataFrame] = None,
     row: Optional[Dict[str, Any]] = None,
     segment_cache: Optional[SegmentCache] = None,
+    trace_id: Optional[str] = None,
```

Comment:
Should we do something with this argument?

Reply:
Good catch, removing.